package/CHANGELOG

```diff
@@ -22,6 +22,7 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
   * 0.21.0
 
 Fixes
+  * Ensures principal_axes() returns axes with the right hand convention (Issue #2637)
```

Comment:
You should add your name/GitHub handle in the `CHANGELOG` as well.

Reply:
Done! Thanks for reminding me.